cmplibrary/src/test/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLibTest.java

```diff
@@ -150,18 +151,19 @@ public void onAction(){
     }
 
     @Test
-    public void onShowOptions() {
+    public void onShowOptions() throws ConsentLibException {
         lib.onShowOptions();
         verify(lib.activity).runOnUiThread(lambdaCaptor.capture());
+        if(lib.onError != null )
```

Comment:
it should not be here

Reply:
@leodoin yes we can remove here , I believe sometimes onError object may be null so added this check